search_engine/src/management/commands/scrape.py

```diff
@@ -0,0 +1,9 @@
+from django.core.management.base import BaseCommand, CommandError
+from .scraper.crawler import crawl
+
+
+class Command(BaseCommand):
+    help = 'Scrape the web'
+
+    def handle(self, *args, **options):
```

Comment:
Missing Docstring :)

Reply:
PEP-8 only mandates docstrings for public-facing functions and classes, so this is a non-issue.